homeassistant/components/deconz/device_trigger.py

```diff
@@ -482,6 +482,12 @@
     (CONF_SHORT_PRESS, ""): {CONF_EVENT: 1002},
 }
 
+LIDL_SILVERCREST_SWITCH_MODEL = "TS004F"
+LIDL_SILVERCREST_SWITCH = {
+    (CONF_SHORT_PRESS, CONF_TURN_ON): {CONF_EVENT: 1002},
+    (CONF_DOUBLE_PRESS, CONF_TURN_OFF): {CONF_EVENT: 1004}
```

Comment:
Copy paste issue?
```suggestion
    (CONF_DOUBLE_PRESS, CONF_TURN_ON): {CONF_EVENT: 1004}
```

Reply:
It's a button.
The manual states that a single press means "turn on" and double press means "turn off", but I guess that doesn't make sense with HA.

Reply:
Well the device trigger will be something like double click "on" button, so it would be weird if the buttons name is different between single and double press